assets/src/edit-story/components/canvas/pagepreview/index.js

```diff
@@ -68,6 +68,7 @@ const PreviewWrapper = styled.div`
   position: relative;
   overflow: hidden;
   background-color: white;
+  border-radius: 4.5px;
```

Comment:
Will we be using the same radius more often? Might be worth putting it into the theme then.

Reply:
Doesn't look like it, this is specific to the small preview